tests/table-operations.spec.ts

```diff
@@ -105,13 +105,21 @@ async function artificialWait(page: Page, tableNumber = 0) {
   await page.getByTestId('btn-page-close').first().click();
 }
 
+const tableName = generateVarName('t');
 test.beforeEach(async ({ page }) => {
   await page.goto('');
 
+  page.on('console', msg => {
+    if (msg.type() === 'error') console.log(`Error text: "${msg.text()}"`);
+  });
```

Comment:
Do we need to do this? It's already logged in the console and can get it from the browser logs

Reply:
No not necessarily I just found it convenient having it show up in your terminal as it goes through the test as well

Reply:
You don't get the browser logs from playwright unless you do this I think